.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+dist: bionic
+language: c
+jobs:
+  include:
+  - os: linux
+    env: CC=clang CXX=clang++ npm_config_clang=1
+    compiler: clang
+cache:
+  directories:
+  - node_modules
+  - "$HOME/.electron"
+  - "$HOME/.cache"
+addons:
+  apt:
+    packages:
+    - libgnome-keyring-dev
+    - icnsutils
+before_install:
+- mkdir -p /tmp/git-lfs && curl -L https://github.com/github/git-lfs/releases/download/v1.2.1/git-lfs-linux-amd64-1.2.1.tar.gz | tar -xz -C /tmp/git-lfs --strip-components 1 && /tmp/git-lfs/git-lfs pull
+- sudo apt-get install --no-install-recommends -y icnsutils graphicsmagick xz-utils
```

Comment:
There is an `apt.packages` section for installing packages, and `icnsutils` is installed twice